Podfile.lock

```diff
@@ -88,4 +88,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 4bdbd04c6f6eb4a3303a81452234849fd67e68b4
 
-COCOAPODS: 1.4.0
+COCOAPODS: 1.4.0.rc.1
```

Comment:
Update cocoapods to `1.5.0`. There is an issue in **Realm** in current version:

> Undefined symbols for architecture x86_64:
> "realm::BpTreeBase::write_subtree(realm::Array const&, unsigned long, unsigned long, unsigned long, realm::BpTreeBase::SliceHandler&, realm::_impl::OutputStream&)", referenced from:
> realm::BpTree<long long, false>::write(unsigned long, unsigned long, unsigned long, realm::_impl::OutputStream&) const in libRealm.a(RLMResults.o)
> realm::BpTree<long long, false>::write(unsigned long, unsigned long, unsigned long, realm::_impl::OutputStream&) const in librealm.a(RLMResults.o)
> realm::BpTree<long long, false>::write(unsigned long, unsigned long, unsigned long, realm::_impl::OutputStream&) const in libRealm.a(object_store.o)

Reply:
okk @imjog !!